### DISCUSSION TITLE
Possible fixed V2 depth anything model released

### DISCUSSION THREAD

**Discussion by donrikk** [nagadomi/nunif#276] [General]
@nagadomi it seems like this new model which comes in base large small and small-transparent fixes the issues that plague depth anything V2 when it comes too blurred objects and transparent object in source videos. I have not tested it yet but thought I should bring it to ur attention since there are checkpoints out on there huggingface page. I do not think they are diffusion based so it may be something u may be interested in looking at, I hope u have a fantastic holiday season thanks again for all the work u do on this program!
[https://promptda.github.io/](url)

**Comment by loawizard** [comment]
This looks great! Would love to see it implementated if possible

**Comment by nagadomi** [comment]
It is technically impossible regardless of my personal opinion.
Google what LiDAR is.
